bindings/matrix-sdk-ffi/src/event.rs

```diff
@@ -200,7 +200,24 @@ impl TryFrom<AnySyncMessageLikeEvent> for MessageLikeEventContent {
                     in_reply_to_event_id,
                 }
             }
-            AnySyncMessageLikeEvent::RoomRedaction(_) => MessageLikeEventContent::RoomRedaction,
+            AnySyncMessageLikeEvent::RoomRedaction(c) => {
+                let (redacted_event_id, reason) = match c {
```

Comment:
Maybe using `c.redacts(&RoomVersionId)` would be better, but I don't see a way to get this version id from the existing data.

Reply:
Ideally, we'd use this method, but we're not quite guaranteed that we have the data; maybe it's not been requested by the sync, or it's not been received yet.

So I would slightly change the implementation to start with the `content` field, if it's there (should be present for room >= v11), and then use the `redacts` field otherwise (should be present for previous room versions).